device-telemetry/app/com/microsoft/azure/iotsolutions/devicetelemetry/webservice/v1/models/StatusApiModel.java

```diff
@@ -63,45 +62,20 @@ public String getUID() {
     }
 
     @JsonProperty("Properties")
-    public HashMap<String, String> getProperties() {
+    public Dictionary<String, String> getProperties() {
         return this.properties;
     }
 
     @JsonProperty("Dependencies")
-    public HashMap<String, String> getDependencies() {
-        HashMap<String, String> dependencies = new HashMap<>();
-
-        this.statusList.stream()
-            .forEach(s -> {
-                String dependencyStatusMessage = s.isHealthy() ? "OK" : "ERROR";
-                dependencies.put(s.getName(), dependencyStatusMessage + ": " + s.getStatusMessage());
-            });
-        return dependencies;
+    public Dictionary<String, StatusResultApiModel> getDependencies() {
+        return this.dependencies;
     }
 
     @JsonProperty("$metadata")
-    public HashMap<String, String> getMetadata() {
-        return new HashMap<String, String>() {{
-            put("$type", "Status;" + Version.NAME);
-            put("$uri", "/" + Version.NAME + "/status");
+    public Dictionary<String, String> getMetadata() {
```

Comment:
why change HashMap to Dictionary?

Reply:
It was dictionary in every other service and hence for consistency.